$NetBSD: patch-meson.build,v 1.1 2021/12/02 07:28:40 pho Exp $

Impedance adjustment with librefuse. Never send this to the
upstream. It's our code that needs to be fixed, not theirs!

TODO: This patch should *really* be gone. Update librefuse.

--- meson.build.orig	2021-12-01 17:06:20.555815467 +0000
+++ meson.build
@@ -44,7 +44,8 @@ endif
 configure_file(output: 'config.h',
                configuration : cfg)
 
-sshfs_deps = [ dependency('fuse3', version: '>= 3.1.0'),
+sshfs_deps = [ dependency('fuse3', version: '>= 3.1.0', required: false),
+               dependency('fuse' , version: '>= 2.6.0', required: false),
                dependency('glib-2.0'),
                dependency('gthread-2.0') ]
 
